charts/stash-postgres/templates/postgres-backup-function.yaml

```diff
@@ -19,6 +19,7 @@ spec:
   - --enable-cache=${ENABLE_CACHE:=true}
   - --max-connections=${MAX_CONNECTIONS:=0} # 0 indicates use default connection limit
   - --hostname=${HOSTNAME:=}
+  - --pg-backup-cmd=${pgBackupCMD:=} # can specify dump command with either pg_dump or pg_dumpall
```

Comment:
Can we rename it to only `backup-cmd` and default it from `.Values.backup.cmd`? This will help the user to set the command during installing the addon and he won't need to provide the variable in every `BackupConfiguration`.

Reply:
I agree. I will change it